packages/components/data-resource/s3-endpoint-type.tsx

```diff
@@ -110,7 +219,48 @@ export const S3EndPointType: React.FC<S3EndpointTypeProps> = (props) => {
           />
         </FormGroup>
       )}
-
```

Comment:
add a StatusBox around "Secret Namaspace", "Secret" dropdown, "Access key" and "Secret key" components (one StatusBox enclosing all) and pass error={false} loading={isClusterAdminLoading || hasReadAccessLoading || hasWriteAccessLoading}... so that we only show these components once we are aware of which user is logged in...

Reply:
not getting why we need a status box here?

Reply:
why need a status box when it can be achieved by this - 

![image](https://user-images.githubusercontent.com/22158580/177350233-785ec9b1-51f5-4156-84dc-db8cf3b67b52.png)

instead of my name all the things which u mentioned

Reply:
It will save us from using `? :` condition, which decreases readability... we can use it if no other option is available.
Make it a good practice to use StatusBox whereever possible, it does lot a more than adding a LoadingBox.